openpype/hosts/fusion/plugins/publish/collect_render.py

```diff
@@ -145,9 +148,11 @@ def get_expected_files(self, render_instance):
         start = render_instance.frameStart - render_instance.handleStart
         end = render_instance.frameEnd + render_instance.handleEnd
 
-        path = (
-            render_instance.tool["Clip"]
-            [render_instance.workfileComp.TIME_UNDEFINED]
+        comp = get_current_comp()
```

Comment:
This should be using the instance that was collect during publishing - which is in `context.data["currentComp"]` as can also be seen on line 38 [here](https://github.com/ynput/OpenPype/pull/5329/files#diff-bd49b6ecadde420bfbddfb8c380da1dc2b8f98a815e57db61204fb4e804670cbR38)